inputgraph/hash_test.go

```diff
@@ -32,7 +32,7 @@ func TestHashTargetWithDocker(t *testing.T) {
 	r.Equal("core", project)
 
 	hex := fmt.Sprintf("%x", hash)
-	r.Equal("e4ccd01c9eeee569917873921a6e5e531c1e4108", hex)
+	r.Equal("8cf1c171937b425cdc926f17eb8ba99aca8d1fff", hex)
```

Comment:
is this an intentional change? I think @mikejholly fixed these in main earlier today.

Reply:
maybe try rebasing against main?

Reply:
I think the hash changed again because the FROM image changed